inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html

```diff
@@ -0,0 +1,168 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2014 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "inspirehep_theme/format/record/Inspire_Default_HTML_general_macros.tpl" import mathjax %}
+
+{% extends config.SEARCH_UI_BASE_TEMPLATE %}
+
+{%- block javascript %}
+  {{ super() }}
+  {{ mathjax() }}
+  {% assets "inspirehep_author_profile_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+{%- endblock javascript %}
+
+{% block body %}
+<div id="record_content" ng-init='record = {{ record|tojson|safe }}'>
+  <div class="record-detailed">
+    <div class="record-header" id="detailed-header">
+      <div class="row">
+        <div class="col-md-12 summary" id="detailed-header-top">
+          <div class="col-md-2">
+            <img src="{{ url_for('static', filename='images/einstein.svg') }}" width="100%">
+          </div>
+          <div class="col-md-10">
+            <div id="record-title">
+              <author-name></author-name>
+              <author-affiliation id="affiliation"></author-affiliation>
+            </div>
+            <div id="fields">
```

Comment:
Add a prefix to all the `ids` to make sure they will not later conflict. Like `author-fields`